src/datanode/src/sql/create.rs

```diff
@@ -249,16 +251,32 @@ impl SqlHandler {
             region_numbers: vec![0],
             primary_key_indices: primary_keys,
             create_if_not_exists: stmt.if_not_exists,
-            table_options: HashMap::new(),
+            table_options,
         };
         Ok(request)
     }
 }
 
+fn stmt_options_to_hashmap(opts: &[SqlOption]) -> error::Result<TableOptions> {
+    let mut map = HashMap::with_capacity(opts.len());
+    for SqlOption { name, value } in opts {
+        let value_str = match value {
+            Value::SingleQuotedString(s) => s.clone(),
+            Value::DoubleQuotedString(s) => s.clone(),
+            _ => value.to_string(),
```

Comment:
Does something strange happen if you give up (just  call toString) dealing with other types?

Reply:
I thought we should only accept quoted string/boolean/numbers as table option, but maybe we should add a table option validator when creating table.